src/Shared/ServerInfrastructure/StringUtilities.cs

```diff
@@ -370,7 +370,7 @@ private unsafe static bool IsValidHeaderString(string value)
             }
         }
 
-        private static readonly char[] s_encode16Chars = "0123456789ABCDEF".ToCharArray();
+        private static ReadOnlySpan<byte> s_hexEncodeMap => new byte[] { (byte)'0', (byte)'1', (byte)'2', (byte)'3', (byte)'4', (byte)'5', (byte)'6', (byte)'7', (byte)'8', (byte)'9', (byte)'A', (byte)'B', (byte)'C', (byte)'D', (byte)'E', (byte)'F' };
```

Comment:
```suggestion
        private static ReadOnlySpan<byte> HexEncodeMap => new byte[] { (byte)'0', (byte)'1', (byte)'2', (byte)'3', (byte)'4', (byte)'5', (byte)'6', (byte)'7', (byte)'8', (byte)'9', (byte)'A', (byte)'B', (byte)'C', (byte)'D', (byte)'E', (byte)'F' };
```
as it is a property? (Though I would be less strict in the case of the ROS-trick, but in other places in this repo it's written as property).